packages/cozy-flags/src/index.js

```diff
@@ -2,11 +2,6 @@
 
 import flag from './flag'
 
-if (typeof window !== 'undefined') {
-  flag.connect = require('./connect').default
-  flag.FlagSwitcher = require('./FlagSwitcher').default
-}
-
```

Comment:
can you use `export default flag` below ? and also remove the typeof check in the other file so that we can have

```
import connect from './connect'

flag.connect = connect
```

Using import and export exclusively is necessary for correct unused code pruning.

Reply:
> remove the typeof check in the other file 

not sure what you mean by that? I assume you mean `index.browser.js` by "other file", maybe there is no `if (typeof window !== 'undefined') {`, if this is what you meant?

Reply:
Ah yes, no if, there is only the "module.exports/require" problem (require should be import/module.exports should be export).

Reply:
Does 044db5e look good?

Reply:
Yep, thanks.